packages/labs/src/labs.md

```diff
@@ -223,3 +225,9 @@ Modifiers are the tool through which you customize Popper.js's behavior. It defi
 <div class="pt-callout pt-intent-primary pt-icon-info-sign">
     To understand all the Popper.js modifiers available to you, you'll want to read [the Popper.js Modifiers documentation](https://popper.js.org/popper-documentation.html#modifiers).
 </div>
+
+@## Tooltip 2.0
```

Comment:
Any docs? Anything? At least explain it's a thin wrapper to add css class and change some defaults